package.json

```diff
@@ -2,13 +2,15 @@
   "name": "govuk-react",
   "version": "0.1.3",
   "dependencies": {
-    "@storybook/react": "^3.2.13",
     "glamorous": "^4.11.2",
     "govuk-colours": "^1.0.2",
     "govuk-elements-sass": "^3.1.2",
+    "prop-types": "^15.6.0",
     "react": "^15.0.0",
     "react-dom": "^15.0.0",
-    "react-scripts": "^1.0.14"
+    "react-scripts": "1.0.14",
+    "enzyme": "^3.3.0",
```

Comment:
enzyme should be a dev dependency - we wouldn't want it installed in production (in fact so should react-scripts and I think govuk-elements-sass)